### PR TITLE
chore: add font and icon instructions to README for project config

### PR DESCRIPTION
**Pull request by stefanoScalzo**
## Related Issue
Closes SAP/fundamental-styles#1758

## Description
Fixes our read me to contain the project configuration as we no longer have the old file.
Also include the font size in px so that they can render the proper rem values

4. Documentation
- [x] Storybook documentation has been created/updated

### CODE REVIEW

**Review by InnaAtanasova — CHANGES_REQUESTED** [review]
What about @sap-theming library? They can add the icons and the fonts from there, can't they?

File: README.md [hunk 1]
```diff
@@ -43,9 +43,31 @@ npm install fundamental-styles --save
...
 The distrubtion folder contains several css files per component: a default file and one file per supported theme (`sap_fiori_3`, `sap_fiori_3_dark`, `sap_fori_3_light_dark`, `sap_fiori_3_hcb`, `sap_fori_3_hcw`). Each themed file includes fallback support for css variables in IE11 for the corresponding theme. The default file includes fallbacks for `sap_fiori_3`. 
 
-### Fonts & Icons
-
-This project does not contain fonts and icons. See our [Getting Started Guide](https://sap.github.io/fundamental-styles/getting-started.html) for more information.
+### Icons
+See the [Icon Component](https://fundamental-styles.netlify.app/?path=/docs/components-icon--sizes) for a list of icon class names. See Project Configuration below for instructions to include SAP Fiori 3 icons in your project.
+
+### Project Confirguration
+This project does not contain fonts and icons - they must be added to your project separately. Download [Font 72](https://experience.sap.com/fiori-design-web/downloads/#download-font-72) and [SAP icons](https://experience.sap.com/fiori-design-web/downloads/#sap-icon-font). After adding fonts and icons to your project, include the following in your css:
```
Comment (JKMarkowski) [inline README.md:50]: This is harder way to make things work. Could you mention there, that user can just download `@sap-theming` and add 
```scss
@font-face {
    font-family: "72";
    src: url("~@sap-theming/theming-base-content/content/Base/baseLib/sap_base_fiori/fonts/72-Regular-full.woff")
        format("woff");
    font-weight: normal;
    font-style: normal;
}
```

and 
```scss
@font-face {
    font-family: "SAP-icons";
    src: url("~@sap-theming/theming-base-content/content/Base/baseLib/sap_fiori_3/fonts/SAP-icons.woff")
        format("woff");
    font-weight: normal;
    font-style: normal;
}
```
for icons

**Comment by InnaAtanasova** [comment]
Please update the title, I think you missed a word

File: README.md (hunk 1 above)
Comment (stefanoScalzo) [reply README.md]: fixed!

**Comment by stefanoScalzo** [comment]
> What about @sap-theming library? They can add the icons and the fonts from there, can't they?

fixed

**Review by InnaAtanasova — APPROVED** [review]

**Review by JKMarkowski — APPROVED** [review]
LGTM

**Pull request merged**